src/singletons/Settings.hpp

```diff
@@ -221,6 +221,8 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting streamerModeMuteMentions = {"/streamerMode/muteMentions", true};
     BoolSetting streamerModeSuppressLiveNotifications = {
         "/streamerMode/supressLiveNotifications", false};
+    BoolSetting streamerModeSuppressModActions = {
+        "/streamerMode/streamerModeSuppressModActions", true};
```

Comment:
why is `streamerMode` twice there, where's consistency? Everything around these lines of code has it just as a category, not also as a prefix later on.

Reply:
I think my completion changed this and I didn't notice. Fixed in [fbe9799](https://github.com/Chatterino/chatterino2/pull/3926/commits/fbe97999e96f3e24518bce31c73999c5b59c7450).